packages/scripts/test/application.unit.ts

```diff
@@ -271,6 +271,22 @@ describe('Application', function () {
             expect(faviconHref).to.equal('favicon.ico');
         });
 
+        it('launches a built application with web environment and given metadata', async () => {
```

Comment:
maybe repeat this test also for engineer, where we test the run command

Reply:
added